src/main/java/Duke.java

```diff
@@ -6,5 +8,55 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        printGreetings();
+        storeTasks();
+    }
+
+    public static void printGreetings() {
```

Comment:
It would be more appropriate to name the method as 'printGreeting' instead of 'printGreetings' since the method only contains one greeting.